java/CONTRIBUTING.rst

```diff
@@ -0,0 +1,38 @@
+Bulding the Java Cookbook
+=========================
+
+The Java cookbook uses the Sphinx documentation system.
+
+Running ``make java`` from the cookbook root directory (the one where
+the ``README.rst`` exists) will install all necessary dependencies
+and will compile the cookbook to HTML.
+
+You will see the compiled result inside the ``build/java`` directory.
+
+Testing Java Recipes
+====================
+
+All recipes in the cookbook must be tested. The cookbook uses
+``javadoctest`` to verify the recipes.
```

Comment:
We should explain here that we need at least JDK9 for JShell.

Reply:
Added this detail to CONTRIBUTING.rst document